server/dash/dashboards/neighborhood_recent.py

```diff
@@ -259,12 +296,20 @@ def update_figure(selected_council):
     Input("council_list", "value")
 )
 def update_council_figure(selected_council):
-    """Dash Callback Function to update the Request Type Pie Chart at the middle of the dashboard.
+    """Updates the Request Type Pie Chart based on data filtered by selected_council.
 
     This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
     outputs the the pie chart showing the share of each request types. The pie chart will
     show up inside dcc.Graph object as long as id "pie_graph" is passed in.
 
+    Args:
+        selected_council:
+            A string argument automatically detected by Dash callback function when "council_list" element is selected in the layout.
+    
+    Returns:
+        pie_graph
```

Comment:
for `Returns`, we don't need to name the return value. So you can delete line 310 here.

Reply:
Resolved